src/blockchain_utilities/blockchain_prune.cpp

```diff
@@ -601,26 +646,27 @@ int main(int argc, char* argv[])
   MDB_env *env0 = NULL, *env1 = NULL;
   open(env0, paths[0], db_flags, true);
   open(env1, paths[1], db_flags, false);
-  copy_table(env0, env1, "blocks", MDB_INTEGERKEY, MDB_APPEND);
-  copy_table(env0, env1, "block_info", MDB_INTEGERKEY | MDB_DUPSORT| MDB_DUPFIXED, MDB_APPENDDUP, BlockchainLMDB::compare_uint64);
+  copy_table(env0, env1, "blocks", MDB_INTEGERKEY, 0);
+  copy_table(env0, env1, "block_info", MDB_INTEGERKEY | MDB_DUPSORT| MDB_DUPFIXED, 0, BlockchainLMDB::compare_uint64);
   copy_table(env0, env1, "block_heights", MDB_INTEGERKEY | MDB_DUPSORT| MDB_DUPFIXED, 0, BlockchainLMDB::compare_hash32);
   //copy_table(env0, env1, "txs", MDB_INTEGERKEY);
-  copy_table(env0, env1, "txs_pruned", MDB_INTEGERKEY, MDB_APPEND);
-  copy_table(env0, env1, "txs_prunable_hash", MDB_INTEGERKEY | MDB_DUPSORT | MDB_DUPFIXED, MDB_APPEND);
```

Comment:
If you're copying something in DB order, in this case you could use both MDB_APPEND and MDB_APPENDDUP. Their effects are independent, MDB_APPEND applies to the primary key, MDB_APPENDDUP applies only to the dupsort values.

Reply:
OK. The doc implies otherwise: 
> MDB_APPENDDUP - as above, but for sorted dup data
might need rewording then.

Reply:
The implementation seems to handle the case smartly enough though. I.e. if you mdb_cursor_put using MDB_APPENDDUP (on an MDB_DUPSORT db of course), and the key _doesn't_ already exist, it appends a new key and data. If there's an existing key, it appends the new data.